src/routes/(inner)/elements/badges/+page.svelte

```diff
@@ -55,18 +55,18 @@
 	<svelte:fragment slot="usage">
 		<section class="space-y-4">
 			<h2>Icon Badge</h2>
-			<p>Replace <code>.badge</code> with <code>.badge-icon</code> for an icon badge.</p>
+			<p>Replace <code>.badge</code> with <code>.badge-icon</code> for a badge based on an icon. Use the icon library of your choice.</p>
 			<DocsPreview background="neutral">
 				<svelte:fragment slot="preview">
 					<span class="badge-icon variant-filled"><i class="fa-solid fa-skull" /></span>
 				</svelte:fragment>
 				<svelte:fragment slot="source">
-					<CodeBlock language="html" code={`<span class="badge-icon variant-filled">(icon)</i></span>`} />
+					<CodeBlock language="html" code={`<span class="badge-icon variant-filled"> <i class="fa-solid fa-skull" /> </span>`} />
```

Comment:
@oatnog Hey, make sure not to include Font Awesome icons in the code snippets. We avoid this as Skeleton is icon library agnostic. The idea is users will insert the icon of their preference there, so `(icon)` is the preferred way to handle that.